tvtk/pyface/scene_model.py

```diff
@@ -328,6 +333,9 @@ def _get_light_manager(self):
     ######################################################################
     # SceneModel API.
     ######################################################################
+    def _closed_fired(self):
```

Comment:
I do not think there is a need to do this if all you are doing is calling the parent class